jerry-core/ecma/base/ecma-helpers-number.cpp

```diff
@@ -683,7 +683,15 @@ ecma_number_calc_remainder (ecma_number_t left_num, /**< left operand */
 
   ecma_number_t q = ecma_number_trunc (ecma_number_divide (n, d));
 
-  return ecma_number_substract (n, ecma_number_multiply (d, q));
+  ecma_number_t r = ecma_number_substract (n, ecma_number_multiply (d, q));
+
+  if (ecma_number_is_zero (r)
```

Comment:
// https://github.com/Samsung/jerryscript/pull/370#discussion_r34475746

For me, the conditions are not similar enough to combine them to one line, as checks are different and arguments are different.